docs/components/slideout.md

```diff
@@ -77,6 +282,12 @@ There is one default slot which is used to place content into the slideout panel
 </KSlideout>
 ```
 
+- `badgeContent` - used to customize `badge` inside the header
+- `titleContent` - used to customize `title` inside the header
+- `iconContent` - used to customize `icon` inside the header
+
 ## Events
 
-- `@close` - Emitted when the close button is clicked, anything outside the panel is clicked, or the `esc` key is pressed.
+- `close` - Emitted when the close button is clicked, anything outside the panel is clicked, or the `esc` key is pressed.
+
+- `kclipboard-icon-clicked` - Emitted when the `icon` next to `title` is clicked.
```

Comment:
suggestion: if the icon content is slottable we should use a more generic event (if we even need it) - if we allow the consuming app to completely control the content via slots, we no longer need to handle/emit the click event.

Reply:
Yes, removed this.